### ISSUE TITLE
Alt text not clearly distinguishable from body text

### ISSUE THREAD

**Issue by Minnemann-dzblesen** [edrlab/thorium-reader#1983]
During our manual accessibility testing of an EPUB, we encountered the following problem: 
the alt text to an image is well announced with Thorium's integrated read aloud feature. However, it does NOT announce that it is an image. This might be pretty confusing for readers with impairments. The blind reader should recognize that this is an image text.

In combination NVDA or Jaws with Thorium (which is certainly not meant to be), "image" (or also "image end") is announced after or before the alt text in each case. Also the screen reader integrated in Windows or Voice Over with Apple Books announce that it is an image. We have tested with Thorium 2.2.0. 

Or are we doing something wrong?

<figure class="bild simple_container firstInSequence lastInSequence" id="elNr-1-2-2-1-2-10">
		<img src="images/ABB_978-3-498-00115-5_001.jpg" alt="Über der geöffneten Gittertür zu einem Park steht auf einem Metallschild «Parc Simone Veil»."/>
		<figcaption>
				<p class="bild-legende"><aid="crossref-bild-0001"></a>Bild xy: Legendentext für die Abbilung</p>
		</figcaption>
</figure>

**Comment by danielweck** [comment]
Hello, Thorium doesn't inject additional prompts, the TTS readaloud experience is based purely on authored alternate text / accessible descriptions.
If Thorium injected speech prompts at runtime, would they need to match the language of the user, or of the publication metadata, or of the text content itself? I assume the user locale, so there could be a discrepancy with authored language

**Comment by gautierchomel** [comment]
Team discussions so far on the subject:

1. Some users will want additional semantics, but others will prefer a lighter reading experience, meaning that implementation should provide a verbose parameter (as in assistive technologies).
2. If we start with Images we'll have to discuss about Notes, Asides, Emphasizes, etc. 
3. Different levels of semantics will be needed, HTML and ARIA at least (we must be able to differentiate images with roles).
4. Translations will be needed

The unresponded question behind is "shall we consider Thorium TTS feature as an AT". 

On the positive side it would allow for a strong higlight of semantics (meaning that it makes sense to add them for producers) and will certainly serve a lot of users. 

Still, the effort to design, add and maintain is to be considered and dedicated funding shall be found.

**Comment by gautierchomel** [comment]
Also to consider page numbers :) see related issue #1974 and discussions #1951 and #1799.

**Comment by tedvandertogt** [comment]
I remember from narrating books as a volunteer, that we had a whole set of rules including how to handle special elements (like authors, page number, images, footnotes),  If a user relies on audio alone, he or she will easily get confused without those prompts. Depending on the type of user, the desirable UX may differ. I suggest to draft user requirements for several user types, and decide how to address these without introducing too much complexity

**Comment by Minnemann-dzblesen** [comment]
Many thanks for the numerous replies, the good discussion and thoughts on this and even if there is no prompt solution to this, the drive that comes in here. We as a German institution and also many other European institutions often and primarily recommend Thorium to publishers (to check their EPUBs in fact of the approaching EAA), so a comprehensible and high quality is important to us, but I am sure about that in the process.
